### PR TITLE
Collection contacts

### PR DESCRIPTION
**Pull request by tyom**
- Add collection to `/contacts` index page
- Add collection to company contacts page

### All contacts
all-contacts

### Company contacts
company-contacts

### CODE REVIEW

File: assets/stylesheets/application.scss [hunk 1]
```diff
@@ -2,9 +2,6 @@ $images-dir: "/images/";
 
 @import "_deprecated/trade-elements";
 
-$fa-font-path: "~font-awesome/fonts";
```
Comment (ztolley) [inline assets/stylesheets/application.scss]: Not sure why this couldn't have been done in it's own PR

File: src/apps/contacts/middleware.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  const selectedFiltersQuery = pick(req.query, [
+    'sector',
+    'company',
+  ])
+
+  const selectedSortBy = req.query.sortby ? { sortby: req.query.sortby } : null
+
+  req.body = Object.assign({}, req.body, selectedSortBy, pickBy(selectedFiltersQuery))
+
+  console.log(req.body)
```
Comment (ztolley) [inline src/apps/contacts/middleware.js]: naughty

File: src/apps/contacts/middleware.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+const { searchContacts } = require('../search/services')
+const { transformContactsResultsToCollection } = require('./transformers')
+
+async function getContactsCollection (req, res, next) {
+  try {
+    res.locals.results = await searchContacts({
+      token: req.session.token,
+      requestBody: req.body,
+      limit: 10,
```
Comment (ztolley) [inline src/apps/contacts/middleware.js]: Is this 10 and the 10 below related?

File: src/apps/contacts/transformers.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  archived_on,
+  archived_reason,
+  company_sector,
+} = {}) {
+  if (!id || !first_name || !last_name) { return }
+
+  const item = {
+    id,
+    type: 'contact',
+    name: `${first_name} ${last_name}`,
```
Comment (ztolley) [inline src/apps/contacts/transformers.js]: May need to trim for when someone only enters a last name

File: src/apps/contacts/transformers.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        value: get(company, 'name'),
+      },
+      {
+        label: archived_on ? 'Archived' : 'Updated',
+        type: 'datetime',
+        value: archived_on || modified_on,
+      },
+      {
+        label: 'Sector',
+        type: 'datetime',
```
Comment (ztolley) [inline src/apps/contacts/transformers.js]: datetime?

File: src/apps/contacts/transformers.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  if (!id || !first_name || !last_name) { return }
+
+  const item = {
+    id,
+    type: 'contact',
+    name: `${first_name} ${last_name}`,
+    isArchived: archived,
+    meta: [
+      {
+        label: 'Company',
```
Comment (ztolley) [inline src/apps/contacts/transformers.js]: Would be nicer if labels were taken from a label file.

File: assets/stylesheets/application.scss (hunk 1 above)
Comment (tyom) [inline assets/stylesheets/application.scss]: It's relevant to contacts as it was the only place those icons were used, AFAIK.

File: src/apps/contacts/middleware.js (hunk 3 above)
Comment (tyom) [inline src/apps/contacts/middleware.js]: The ten below is `parseInt` [radix](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Parameters) which stands for decimal numeral system.

File: src/apps/contacts/middleware.js (hunk 2 above)
Comment (tyom) [inline src/apps/contacts/middleware.js]: Good spot.

File: src/apps/contacts/transformers.js (hunk 4 above)
Comment (tyom) [inline src/apps/contacts/transformers.js]: 👍

File: src/apps/contacts/transformers.js (hunk 6 above)
Comment (tyom) [inline src/apps/contacts/transformers.js]: Yes, but the label file needs to be rewritten to make consistent with investment projects. Maybe another PR.

File: src/apps/contacts/middleware.js (hunk 3 above)
Comment (ztolley) [inline src/apps/contacts/middleware.js]: yeah sorry, me being stupid
